packages/semantic-ui/test/util.test.js

```diff
@@ -0,0 +1,99 @@
+import { getSemanticErrorProps, getSemanticProps } from '../src/util';
+
+
+describe("util js functions", () => {
+
+  describe("getSemanticProps", () => {
+
+      test("defaultSchemaProps should be returned", () => {
```

Comment:
Actually, just one more thing. Can you add a snapshot test where we add a single semantic-specific prop to override an input field, then render a `<Form>` element, then the snapshot should ensure that the form is not overridden? This way we test the full end-to-end use case for #2619.

Reply:
@epicfaace test updated